packages/runtime/src/runtime.ts

```diff
@@ -264,6 +281,8 @@ export class Runtime {
     this.ctx.state.globalApps.delete(0); // remove zero app from context after execution
     senderAcc.createdApps.delete(0); // remove zero app from sender's account
     senderAcc.createdApps.set(this.appCounter, attributes);
+    // save programs in programMap
+    this.programMap.set(this.appCounter, [approvalProgram, clearProgram]);
```

Comment:
Can we use an object instead of array to represent approval and clear programs?
Eg `appId => {approval: approvalProgram, clear: clearProgram}` instead of `[approval, clear]`